docker/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Copyright 2022, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+case "$(uname -m)" in
+    arm64)
+        DOCKER="docker buildx build --platform linux/amd64"
+        ;;
+    *)
+        DOCKER="docker build"
+        ;;
+esac
+
+$DOCKER $@
```

Comment:
Do `exec $DOCKER "$@"`
The double quotes are needed.  And exec means you don't waste a fork/process.

Reply:
For the `arm64` case statement see my other comment for @axel-h -- I didn't want to restrict this to Darwin, because it doesn't really have anything to do with the M1 or the host OS as such. It's just about the architecture of the host OS, which determines the architecture of the docker build.

Wether `buildx` is available or not mostly depends on how recent the host OS is and what the user has installed. But that's also true for just `docker`.